### PR TITLE
kotsadm-postgres statefulset

### PR DESCRIPTION
**Pull request by areed**

### CODE REVIEW

File: addons/kotsadm/alpha/install.sh [hunk 1]
```diff
@@ -81,17 +82,24 @@ function kotsadm_outro() {
     printf "\n"
 }
 
-function kotsadm_secret_auto_create_cluster_token() {
-    local AUTO_CREATE_CLUSTER_TOKEN=$(kubernetes_secret_value default kotsadm-auto-create-cluster-token token)
+function kotsadm_secret_cluster_token() {
+    local CLUSTER_TOKEN=$(kubernetes_secret_value default kotsadm-cluster-token token)
```
Comment (laverya) [inline addons/kotsadm/alpha/install.sh]: the field that it's stored under is `kotsadm-cluster-token` too

File: addons/kotsadm/alpha/tmpl-secret-cluster-token.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: kotsadm-auto-create-cluster-token
+  name: kotsadm-cluster-token
   labels:
     kots.io/kotsadm: 'true'
 stringData:
-  token: ${AUTO_CREATE_CLUSTER_TOKEN}
+  token: ${CLUSTER_TOKEN}
```
Comment (laverya) [inline addons/kotsadm/alpha/tmpl-secret-cluster-token.yaml]: the key and the secret name are both `kotsadm-cluster-token` in kots
```suggestion
  kotsadm-cluster-token: ${CLUSTER_TOKEN}
```

File: scripts/Manifest [hunk 3]
```diff
@@ -5,7 +5,7 @@ ROOK_VERSION=1.0.4
 CONTOUR_VERSION=0.14.0
 REGISTRY_VERSION=2.7.1
 PROMETHEUS_VERSION=0.33.0
-KOTSADM_VERSION=1.12.1
+KOTSADM_VERSION=alpha
```
Comment (laverya) [inline scripts/Manifest]: was this supposed to change? What does it affect?

I just want to make sure we don't make alpha the default somehow

Comment (areed) [reply scripts/Manifest]: This is used in dev to select addons without having to run the kurl web server.

File: addons/kotsadm/alpha/install.sh (hunk 1 above)
Comment (areed) [reply addons/kotsadm/alpha/install.sh]: fixed

**Pull request merged**